changelog/_unreleased/2025-02-15-prevent-imitating-when-not-possible.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Prevent imitating as customer when not possible
+issue: NEXT-40673
+author: Melvin Achterhuis
+author_email: melvin@achterhuis.work
+author_github: @MelvinAchterhuis
+---
+# Administration
+* Added customer active check in `computed` for `src/Administration/Resources/app/administration/src/module/sw-customer/component/sw-customer-card/index.js`.
+* Added new `computed` called `customerImitationWarning` to determine the warning message in `src/Administration/Resources/app/administration/src/module/sw-customer/component/sw-customer-card/index.js`.
+* Changed logic in for tooltip message and disabled in `src/Administration/Resources/app/administration/src/module/sw-customer/component/sw-customer-card/sw-customer-card.html.twig`.
+* Added separate snippets for each scenario in for `module/sw-customer`.
```

Comment:
There's no need to mention snippet changes, or to go into detail. A one-liner like this will be enough:
```suggestion
* Changed `src/module/sw-customer/component/sw-customer-card` to show detailed hints on disabled imitate customer button.
```